index.ts

```diff
@@ -82,13 +82,77 @@ function makeHash(viewFunction: string): string {
     return createHash("MD5").update(viewFunction).digest("hex");
 }
 
+export interface ViewFunctionConfig {
+    viewFunction: string;
+    model?: any;
+    decoder?: string;
+    indent?: number;
+    newLines?: boolean;
+}
+
+// compiles multiple view functions into one elm file
+// which is much faster if you're likely to need all of them
+export function grouped(
+    rootDir: string, moduleName: string, configs: ViewFunctionConfig[],
+    alreadyRun?: boolean, elmMakePath?: string, installMethod?: string): Promise<string[]> {
+    const moduleHash = makeHash(moduleName);

```

Comment:
💭 why does it need a module hash? Why do you need to say which module the functions are in? Don't we already know the module name from the `viewFunction`?